README.md

```diff
@@ -9,9 +9,17 @@
 
 *Particle\Validator* is a very small validation library, with the easiest and most usable API we could possibly create.
 
+## Install
+
+```bash
```

Comment:
This is nice, but maybe also add a small text: "To easily include *Particle\Validator* into your project, install it via [composer](https://getcomposer.org/) using the command line:"